include/openenclave/internal/report.h

```diff
@@ -47,6 +47,57 @@ typedef struct _oe_get_qe_identity_info_args
     uint8_t* host_out_buffer; /* out */
 } oe_get_qe_identity_info_args_t;
 
+/*
+**==============================================================================
+**
+** oe_collaterals_header_t
+**
+**==============================================================================
+*/
+typedef struct _oe_collaterals_header
+{
+    /** Size of the collaterals */
+    uint32_t collaterals_size;
+
+    /** Collaterals data **/
+    uint8_t collaterals[];
+
+} oe_collaterals_header_t;
+
+OE_STATIC_ASSERT(sizeof(oe_collaterals_header_t) == 4);
+
+/*
+**==============================================================================
+**
+** oe_collaterals_t
+**
+** Structure with the collateral contents.  The collaterals are used during
+** the verification of the oe_report_t.
+**
+**==============================================================================
+*/
+typedef struct _oe_collaterals
+{
+    oe_get_qe_identity_info_args_t qe_id_info;
```

Comment:
isn't "qe" an SGX-only term?   We shouldn't have SGX-only names in a non-SGX-specific file.

Reply:
Agreed.  There are plans to change this struct when we make this public.  I opened #2206 for exposing the collaterals API in general.